tests/Admin/ArticleCategoryAdminTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Tests\AppBundle\Admin;
+
+use AppBundle\DataFixtures\ORM\LoadAdminData;
+use Symfony\Component\DomCrawler\Form;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\ControllerTestTrait;
+use Tests\AppBundle\MysqlWebTestCase;
+
```

Comment:
Please add a `@functional` tag for functional tests.
see https://github.com/EnMarche/en-marche.fr/blob/develop/Makefile#L116